config/sync/core.extension.yml

```diff
@@ -7,13 +7,18 @@ module:
   dblog: 0
   dynamic_page_cache: 0
   editor: 0
+  entity_reference_revisions: 0
   field: 0
+  field_ui: 0
```

Comment:
`field_ui` is a developer module and should not be added to the default config (we may solve this when/if we remove the default config).